.ado/pipelines/templates/jobs-ui-app-build.yaml

```diff
@@ -17,16 +17,17 @@ jobs:
     displayName: 'NPM install'
     retryCountOnTaskFailure: 1
     inputs:
-      command: 'install'
-      workingDir: ${{ parameters.workingDirectory }}
+      command: 'custom'
+      customCommand: 'install --prefix .'
```

Comment:
Do you want to add a comment why this is here? It's not super intuitive and I can imagine wondering in 6 months what we did here... :D

Reply:
```suggestion
      customCommand: 'install --prefix .' # overriding prefix to not use the $(prefix) variable
```